content/ko/docs/reference/glossary/sysctl.md

```diff
@@ -0,0 +1,23 @@
+---
+title: sysctl
+id: sysctl
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  유닉스 커널 파라미터를 가져오거나 설정하는데 사용하는 인터페이스
+
+aka:
+tags:
+- 도구
```

Comment:
I think we leave these as-is.

See https://kubernetes.io/ko/docs/reference/glossary/

(we should localize the display titles for these tags, mind)

Reply:
@sftim Thank you! I applied.